packaging/src/docker/Dockerfile

```diff
@@ -14,14 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+ARG BUILD_ENV
+
 FROM ubuntu as unarchive
```

Comment:
Cloud you please add ARG `BUILD_ENV=unarchive` to the `build.sh`:
https://github.com/apache/hive/blob/master/packaging/src/docker/build.sh#L115

Reply:
Done